packages/create-cloudflare/templates/hello-world-durable-object/js/src/index.js

```diff
@@ -16,25 +18,27 @@
  */
 
 /** A Durable Object's behavior is defined in an exported Javascript class */
-export class MyDurableObject {
+export class MyDurableObjec extends DurableObject {
 	/**
 	 * The constructor is invoked once upon creation of the Durable Object, i.e. the first call to
-	 * 	`DurableObjectStub::get` for a given identifier
+	 * 	`DurableObjectStub::get` for a given identifier (no-op constructors can be omitted)
```

Comment:
We say no op constructors can be omitted, but then include one in the example?

Reply:
Yeah. I think that it's helpful to include by default but note that it isn't explicitly required.